infrastructure/common/teams.ts

```diff
@@ -14,6 +14,7 @@ interface Team {
   name: string;
   // Must match "name" in the Digital Land dataset "local-authority-district" (https://www.planning.data.gov.uk/dataset/local-authority-district)
   uniformInstances?: string[];
+  stagingOnly?: boolean;
```

Comment:
nit: To be totally explicit here we should link this to GovPay, e.g. `govPayStagingOnly`

Reply:
I did consider this although I think as it's at a top level it would skip all secret generation on prod? Although we're only using it at the pay level it's not specific to gov pay? 

I'm happy to go with that though as I guess it does better match how we're using it right now?

Reply:
I think I maybe misunderstood your suggestion, I'll refactor to put it inline with the `uniformInstances` pattern 👍

Reply:
Perfect!